### ISSUE TITLE
Hangfire access with angular routing

### ISSUE THREAD

**Issue by cloud0259** [abpframework/abp#16622]
Hello,

I am trying to access the Hangfire dashboard from the Angular front of my application. I added permissions on Hangfire but I can't access them.
I removed the responseType:'code' in the environment.ts file to have the authentication page in the front.
How can I go about it?
I couldn't find anything about it

I use Abp framework 6

**Comment by gterdem** [comment]
Background job UI should be hosted on your `Http.Api.Host`. Please check https://docs.abp.io/en/abp/latest/Background-Jobs-Hangfire#dashboard-authorization

**Comment by cloud0259** [comment]
Yes this is what I am doing. The problem comes from the fact that the authentication takes place in the front and not in the backend
